src/style-auro-calendar-month.scss

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2022 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Import Auro tokens
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/breakpoints';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
+
+// Support for auroElement styles
+// @import './node_modules/@alaskaairux/webcorestylesheets/dist/auroElement/auroElement';
+
+// container styles - defines the look of the outer custom component
+// :host {
+//   ...
+// }
+
+/* stylelint-disable selector-class-pattern, property-no-vendor-prefix, order/properties-order, no-descending-specificity, color-function-notation, max-line-length */
+
+:host {
+  display: block;
+  position: relative;
+
+  width: 336px; // is this right?
+
+  padding: var(--auro-size-200);
+
+  background-color: var(--auro-color-background-lightest);
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 0 0 var(--auro-size-50);
+  text-align: center;
+  color: var(--range-datepicker-month-text);
+  height: 44px;
```

Comment:
That's an odd number? Can we make this `var(--auro-size-600)`?

Reply:
It's odd but it's what makes the layout work for design from Kayla. Any opinion to add here, @leeejune?